docs/source/release/v3.14.0/framework.rst

```diff
@@ -55,18 +55,31 @@ Python
 New
 ###
 
- - New python validator type: `:class:`~mantid.geometry.OrientedLattice`. Checks whether a workspace has an oriented lattice object attached.
+- New python validator type: `:class:`~mantid.geometry.OrientedLattice`. Checks whether a workspace has an oriented lattice object attached.
+
+- As part of the initiative to move towards ``Instrument 2.0`` new objects have been exposed to Python and are now available via workspaces. The newly exposed classes include:
```

Comment:
Users won't know what "Instrument 2.0" means. Maybe better to say: "We have been making major performance improvements to geometry access in Mantid over the last few releases. We are now exposing these features via python to give our users direct access to the same benefits as part of their scripts."